WalletWasabi.Fluent/Helpers/TileHelper.cs

```diff
@@ -9,90 +9,62 @@ namespace WalletWasabi.Fluent.Helpers;
 
 public static class TileHelper
 {
-	public static List<TileViewModel> GetNormalWalletTiles(WalletViewModel walletVm)
+	public static List<TileViewModel> GetWalletTiles(WalletViewModel walletVm, bool isWatchOnly, bool hasBalance)
```

Comment:
Both invocations to `GetWalletTiles()` pass `wallet.KeyManager.IsWatchOnly` for the second parameter, but you could grab that information from `walletVm.Wallet.KeyManager.IsWatchOnly` instead, obviating the need for the second parameter.

In fact I would prefer this method to just receive `WalletViewModel` as a single parameter and infer all other information from there.

This way you're passing full objects as parameters instead of loose primitive values, and leaving the logic/responsibility of "reading" those objects inside the method itself rather than its consumers. This is a general practice that IMO should be used whenever possible.

Reply:
makes sense :+1: